### PR TITLE
[REVIEW] Fix cybert streamz dask startup

### PR DESCRIPTION
**Pull request by brhodes10**
* Fix to startup one dask worker per gpu
* Minor fix to cybert script
* Update to README

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by brhodes10** [comment]
rerun tests

**Comment by brhodes10** [comment]
rerun tests

**Review by BartleyR — APPROVED** [review]

File: examples/streamz/README.md [hunk 1]
```diff
@@ -30,13 +30,13 @@ Create a new container using the image above. When running this container, it wi
 
 ##### Preferred - Docker CE v19+ and nvidia-container-toolkit
 ```
-docker run -it --gpus '"device=0,1,2"' -p 8787:8787 -v /path/to/dataset:/path/to/dataset -v /path/to/model.pth:/path/to/model.pth -v /path/to/label.txt:/path/to/label.txt --name cybert-streamz -d cybert-streamz:latest \
+docker run -it --gpus '"device=0,1,2"' -p 8787:8787 -v /path/to/dataset:/path/to/dataset -v /path/to/model.pth:/path/to/model.pth -v /path/to/label.txt:/path/to/label.json --name cybert-streamz -d cybert-streamz:latest \
```
Comment (bsuryadevara) [inline examples/streamz/README.md:33]: Would you like to add a note `cuda visible devices` id's to be the same or a subset of `device`?

**Review by bsuryadevara — COMMENTED** [review]
Looks good

File: examples/streamz/README.md (hunk 1 above)
Comment (brhodes10) [reply examples/streamz/README.md:33]: Thanks! I have added the explanation in the parameter section.

**Review by bsuryadevara — APPROVED** [review]

**Review by efajardo-nv — APPROVED** [review]

**Pull request merged**